File: C/1010.c
#include <stdio.h>

int main ()
{
    double A, B, C, D, E, F, TOTAL;
    scanf ("%lf %lf %lf", &A, &B, &C);
    scanf ("%lf %lf %lf", &D, &E, &F);
    TOTAL = B * C + E * F;
    printf ("VALOR A PAGAR: R$ %.2lf\n", TOTAL);
    return 0;
}

File: JavaScript/1010.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const firstItem = lines.shift().split(' ');
const secondItem = lines.shift().split(' ');

var idFirstItem = parseInt(firstItem.shift());
var qtdFirstItem = parseInt(firstItem.shift());
var valueFirstItem = parseFloat(firstItem.shift());

var idSecondItem = parseInt(secondItem.shift());
var qtdSecondItem = parseInt(secondItem.shift());
var valueSecondItem = parseFloat(secondItem.shift());

const total = ((qtdFirstItem*valueFirstItem) + (qtdSecondItem*valueSecondItem)).toFixed(2);

console.log("VALOR A PAGAR: R$ " + total);
